### DISCUSSION TITLE
Unlocking features that are not on the free plan when self-hosting

### DISCUSSION THREAD

**Discussion by levrik** [Flagsmith/flagsmith#3418] [Q&A] [answered]
I'm wondering about features like "Change Requests" which on Cloud are not available on the free plan.
On my self-hosted test instance I have enabled the `4eyes` feature flag but it does nothing effectively. I also ensured that it gets returned to the UI.
After checking the frontend code it skips the feature flag check completely if the logged in user is on the free plan which is what the API returns to the UI here.

Now I wonder: Is this intended that these features cannot be used in the open-source version OOTB, meaning requiring manual code changes? Or am I missing something here to get such features enabled?
Given the license and open source FAQ on Flagsmith's homepage I got the impression that everything that's in this repo is free to use when self-hosting.

**Comment by dabeeeenster** [comment, marked as answer]
We describe the differences between the versions here: https://docs.flagsmith.com/version-comparison

The UI will skip over items that might not be present in either the UI or the API with regards to the closed-source features, so it is behaving as intendend, yes.

**Reply by levrik** [reply]
I somehow missed that page, sorry for that. Since I saw a directory in the source code for change requests, I assumed the feature is part of the open source version. After closer inspection now I realized that it just seems to contain some glue code between the open source version and some proprietary code not included in the repo.

All clear now!
